lib/kasket/query_parser.rb

```diff
@@ -12,8 +12,9 @@ class QueryParser
     def initialize(model_class)
       @model_class = model_class
 
-      @supported_query_pattern = /^select\s+((?:`|")#{@model_class.table_name}(?:`|")\.)?\* from (?:`|")#{@model_class.table_name}(?:`|") where (.*?)(|\s+limit 1)\s*$/i
+      @supported_query_pattern = /^select\s+(.+?)\s+from (?:`|")#{@model_class.table_name}(?:`|") where (.*?)(|\s+limit 1)\s*$/i
 
+      @star_pattern = /^((?:`|")#{@model_class.table_name}(?:`|")\.)?\*$/
```

Comment:
This would be slightly better if the first noncapturing group captured and you used `\1` for the second noncapturing group. Ensures matching delimiters.